modules/dnn/test/test_tf_importer.cpp

```diff
@@ -302,17 +302,18 @@ TEST(Test_TensorFlow, defun)
 
 TEST(Test_TensorFlow, fp16)
```

Comment:
Can we make this test parametric and add to `Test_TensorFlow_layers` group?

Reply:
I changed the fp16 test, make it use  DNN_TARGET as the parameter.